modules/hcp-mce-acm-relationship-intro.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-mce-acm-dependency-intro_{context}"]
+= Relationship between {hcp}, {mce-short}, and {rh-rhacm}
+
+You can configure {hcp} by using the {mce}. The multicluster engine is an integral part of {rh-rhacm-first} and is enabled by default. {rh-rhacm} cluster lifecycle defines the process of creating, importing, managing, and destroying Kubernetes clusters across various infrastructure cloud providers, private clouds, and on-premises data centers.
```

Comment:
...and is enabled by default **with {rh-rhacm}**. {mce-short} cluster lifecycle defines the process ...

Reply:
Thanks for the review! Addressed.